mne/gui/_ieeg_locate_gui.py

```diff
@@ -0,0 +1,859 @@
+# -*- coding: utf-8 -*-
+"""Intracranial elecrode localization GUI for finding contact locations."""
+
+# Authors: Alex Rockhill <aprockhill@mailbox.org>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+import numpy as np
+from functools import partial
+
+import matplotlib.pyplot as plt
```

Comment:
You should not do this if possible because it will cause matplotlib to instantiate its GUI backend. If you refactor the creation below you end up not needing it anyway.